cypress/e2e/userIsAbleToSearch.cy.js

```diff
@@ -62,4 +62,15 @@ describe("Search", () => {
 		cy.wait(2000);
 		cy.get("div").should("contain", "Failed to search place by text, 'Text' must have length at least 1");
 	});
+
+	it("should enable nl search and allow user to view POI after nl search", { scrollBehavior: false }, () => {
+		cy.get('[class="amplify-label amplify-switch__wrapper amplify-switch__wrapper--end"]').click();
+		cy.get('[placeholder="Search"]').click();
+		cy.wait(2000);
+		cy.get('[inputmode="search"]').type("Show me three pizza places near Eiffel Tower?").wait(5000).type("{enter}");
+		cy.wait(15000);
```

Comment:
Is there a way we can be a little nicer here?

As in can we wait up to 15 seconds maximum for results to show up over always waiting 15 seconds.

Reply:
Couldn't find an explicit way to do this, but will update it later when I find something that will result in that functionality.